apps/pie-microsite/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.31.0
```

Comment:
Will need to bump this version – @jamieomaguire as this is a small change, is it worth making this the next minor version and just getting it merged?